docs/content/preview/migrate/known-issues.md

```diff
@@ -536,13 +952,100 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
 An example schema on the source database is as follows:
 
 ```sql
-create index ON timestamp_demo (ts);
+CREATE INDEX ON timestamp_demo (ts);
 ```
 
-Suggested change to the schema is to add the `asc` clause as follows:
+Suggested change to the schema is to add the `ASC` clause as follows:
 
 ```sql
-create index ON timestamp_demo (ts asc);
+CREATE INDEX ON timestamp_demo (ts ASC);
 ```
 
 ---
+
+#### Exporting data with names for tables/functions/procedures using special characters fails
+
+**GitHub links**: [Issue #636](https://github.com/yugabyte/yb-voyager/issues/636), [Issue #688](https://github.com/yugabyte/yb-voyager/issues/688), [Issue #702](https://github.com/yugabyte/yb-voyager/issues/702)
+
+**Description**: If you define complex names for your source database tables/functions/procedures using backticks or double quotes for example, \`abc xyz\` , \`abc@xyz\`, or "abc@123", the migration hangs during the export data step.
+
+**Workaround**: Rename the objects (Tables/functions/procedures) on the source database to something without special characters.
+
+**Example**
+
+An example schema on the source PostgreSQL database is as follows:
+
+```sql
+CREATE TABLE `xyz abc`(id int);
+INSERT INTO `xyz abc` VALUES(1);
+INSERT INTO `xyz abc` VALUES(2);
+INSERT INTO `xyz abc` VALUES(3);
+```
+
+The exported schema is as follows:
+
+```sql
+CREATE TABLE "xyz abc" (id bigint);
+```
+
+The preceding example will result in an error as follows:
```

Comment:
These are our backend logs. The user doesn't need to go into such details. We may just mention something generic like it'll throw an error or hang.